_data/master/navbars/usage.yml

```diff
@@ -7,14 +7,18 @@ toc:
   path: /usage/
 - title: Calico and Systemd
   path: /usage/configuration/as-service
-- title: BGP Peering
+- title: Configuring BGP Peers
   path: /usage/configuration/bgp
 - title: Route Reflector
   path: /usage/bird-rr-config
 - title: IPv6
   path: /usage/ipv6
 - title: External Connectivity
   path: /usage/external-connectivity
+- title: Configuring IP-in-IP
+  path: /usage/configuration/ip-in-ip
+- title: Configuring a node IP address and subnet
```

Comment:
tee tee yes we probably should.

I prefer Sentence Case to be honest, but I think I'm not on the winning team there - so i'll change to CMS style.